.gitignore

```diff
@@ -15,3 +15,6 @@ docs/.vitepress/dist
 docs/.vitepress/cache
 
 Gemfile.lock
+
+# JetBrains
+.idea
```

Comment:
imo, IDE-specific ignores work better in the global git config. In other cases OSS projects get invaded by different IDE users 😅

Reply:
Haha, I am keeping changing work machine so I am a bit lazy to configure those global settings. Properly it's time to unwind that bad habit then.

Reply:
@skryukov I ported back to `inertia-rails` at:
* https://github.com/inertiajs/inertia-rails/pull/174

I am going to close this PR to avoid double handling.